src/frontend/src/lib/types/oisyDappDescription.ts

```diff
@@ -27,14 +27,19 @@ const dAppDescriptionSchema = z.object({
 	video: z.string().optional(),
 	videoContentType: z.enum(['video/webm', 'video/mp4']).optional(),
 
-	submittableId: z.string().optional(),
+	submittableId: z.string().optional()
+});
 
-	featured: z.boolean().optional()
+const oisyDappDescriptionSchema = dAppDescriptionSchema.extend({
+	featured: z.boolean().optional(),
```

Comment:
not for this PR but, I guess this can become `true` actually?

Reply:
Sorry ,I don't understand your question.

Reply:
Nvm my comment, my bad.